core-server/src/main/java/org/orbisgis/orbisserver/control/xml/GetCapabilitiesController.java

```diff
@@ -79,18 +79,20 @@
    /** I18N object */
    private static final I18n I18N = I18nFactory.getI18n(IndexController.class);
 
+   private ExceptionType exceptionType = new ExceptionType();
+   private ExceptionReport exceptionReport = new ExceptionReport();
    /**
     * The action method returning the xml file corresponding to the GetCapabilities method. It handles
-    * HTTP GET request on the "/orbisserv/ows" URL. 
+    * HTTP GET request on the "/orbisserv/ows" URL.
     * A good request should be http://localhost:9000/orbisserver/ows?service=WPS&version=2.0.0&request=GetCapabilities
     *
-    * @Parameter the service
-    * @Parameter the version
-    * @Parameter the request GetCpabilities
+    * @Parameter service Name of the service you want to use. Should be WPS here.
+    * @Parameter version Version of the service. It must be an accepted version like 2.0.0.
+    * @Parameter request Request according to the service that you ask to the server. It could be GetCapabilities.
     *
     * @return the xml file
     */
-   @Route(method = HttpMethod.GET, uri = "/orbisserver/ows")
+   @Route(method = HttpMethod.GET, uri = "/orbisserver/wps")
```

Comment:
It was me which requested this change. I just figure out that it is better to keep `ows` instead of `wps`, my bad. @guiMande sorry for the change, could you put back `ows` ?